x-pack/performance/journeys/security_detections_dashboard.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Journey } from '@kbn/journeys';
+import { subj } from '@kbn/test-subj-selector';
+
+export const journey = new Journey({
+  esArchives: ['x-pack/performance/es_archives/security_dashboard_data'],
+})
+  .step('Open Security App and Wait for Fleet Setup', async ({ page, kbnUrl }) => {
+    await page.goto(kbnUrl.get(`/app/security/get_started`));
+    await page.waitForSelector(subj('siem-landing-page'), { timeout: 60000 });
```

Comment:
Looking at the logs it seems like fleet setup is still running in the background when the next step starts. Is there some status API you could use to ensure that setup is complete? I think importing Axios and pinging the kibana server is totally reasonable.

Reply:
Yeah, I was planning to check if the data view was available through the management page but polling the api is probably better than polling the page.